CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.2]
+### Changed
+- **BREAKING:** Removed support for Node v12 in favor of v14 ([#137](https://github.com/MetaMask/eth-json-rpc-middleware/pull/137))
+
```

Comment:
This entry should just go under the `Unreleased` header until we actually cut a release. It's also fine not to modify the changelog at all until a release is cut, because our release automation automatically adds an entry for each commit to the default branch since the previous release.

Reply:
I believe I've changed everything to be correct.